examples/ts_js_interop/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_swc//swc:defs.bzl", "swc")
+load("@bazel_skylib//lib:partial.bzl", "partial")
+
+ts_project(
+    name = "a_b",
+    srcs = [
+        "a.js",
+        "b.ts",
+    ],
+    allow_js = True,
+    declaration = True,
+    emit_declaration_only = True,
+    resolve_json_module = True,
+    transpiler = partial.make(
+        swc,
+        swcrc = ":.swcrc",
+    ),
+    tsconfig = ":tsconfig.json",
+)
+
+filegroup(
+    name = "transpiled_a_b",
```

Comment:
I think these `filegroup` targets are never used.

Can we replace those with `build_test` targets so `bazel test //...` triggers them and they showup in the test results?

Reply:
I'm not familiar with build_test targets, so probably? Have changes gone in that allow the file groups to build successfully?

Reply:
I'm not familiar with build_test targets, but probably? Have changes gone in that allow these filegroups to build successfully now?

Reply:
Search for `build_test` in other tests for examples and try switching these `filegroup` to test targets.

Reply:
https://github.com/aspect-build/rules_ts/blob/3849ce6b8b660e83831c15696f03fd405ffd716e/ts/private/build_test.bzl#L7

I see, makes sense.  It's a trivial test that will always pass as long as it builds.

Reply:
Correct, I've almost exclusively used it for testing rules just like this test you've written.